src/Event/MultiversionManagerEvent.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Drupal\multiversion\Event;
+
+use Drupal\Core\Entity\ContentEntityTypeInterface;
+use Drupal\multiversion\MultiversionMigrationInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * MultiversionManagerEvent class.
+ */
+class MultiversionManagerEvent extends Event {
```

Comment:
Could you please add a short description what the event class is used for?

Reply:
Probably, GitHub just collapsed it above